pkg/kubelet/cm/device_plugin_handler.go

```diff
@@ -291,3 +364,72 @@ func (h *DevicePluginHandlerImpl) readCheckpoint() error {
 	}
 	return nil
 }
+
+func (h *DevicePluginHandlerImpl) removeContents(dir string) error {
```

Comment:
Why isn't this a function is there a need for it to use `DevicePluginHandlerImpl` ?

Reply:
for the usage of checkpointFile(), need to pass in its socketdir.
I feel it weird too to define it as interface of DevicePluginHandlerImpl just because the need of socketdir. Wondering if could pass in socketdir as param

Reply:
Probably a good idea